rts/Sim/Path/HAPFS/PathManager.cpp

```diff
@@ -692,7 +692,7 @@ float3 CPathManager::NextWayPoint(
 	}
 
 	MultiPath* multiPath = localMultiPath.moveDef != nullptr ? &localMultiPath : nullptr;
-	if (multiPath->moveDef == nullptr)
+	if (!multiPath || multiPath->moveDef == nullptr)
```

Comment:
This whole `multiPath` variable looks redundant. How about `if (localMultiPath.moveDef == nullptr)` here and then replace `multiPath->foo` with `localMultiPath.foo` everywhere below?

Reply:
Certainly a good idea, but my intention is to just fix what was giving me a segfault without having to ensure I don't mess anything up in a more substantial refactor.

Reply:
In that case consider at least leaving a ``// FIXME `multiPath` is redundant`` or such